android/pytorch_android/CMakeLists.txt

```diff
@@ -63,13 +81,22 @@ if(APPLE)
         $<TARGET_FILE:pytorch_jni>)
 endif()
 
-target_compile_options(pytorch_jni PRIVATE
-  -fexceptions
+if(BUILD_LITE_INTERPRETER)
```

Comment:
Could those if branches be avoided if to introduce new variable `TARGET`:
```
if(BUILD_LITE_INTERPRETER)
  set(TARGET pytorch_jni_lite)
else()
  set(TARGET pytorch_jni)
endif()
```
and use it:
```
  target_compile_options(${TARGET} PRIVATE
    -fexceptions
  )
  target_include_directories(${TARGET} BEFORE
  PUBLIC $<BUILD_INTERFACE:${libtorch_include_DIR}>
)
```
?

Reply:
Good idea! Address the comment in the latest commit. Btw, https://github.com/pytorch/pytorch/pull/52476 is created to check all CI builds.